### DISCUSSION TITLE
Which version of ios and android react-native-multiple-image-picker supported!

### DISCUSSION THREAD

**Discussion by tuanngocptn** [NitrogenZLab/react-native-multiple-image-picker#150] [Q&A] [answered]
Hello!

I have the plant for init the `react-native-multiple-image-picker` to my project. But I don't know exactly the OS version of ios and android that supported using this library. Anyone can help me.

Thanks

**Comment by baronha** [comment, marked as answer]
> Hello!
> 
> I have the plant for init the `react-native-multiple-image-picker` to my project. But I don't know exactly the OS version of ios and android that supported using this library. Anyone can help me.
> 
> Thanks

This library supports iOS 10+ and `minSdkVersion` for Android: 21
